jwst/master_background/tests/test_master_background_mos.py

```diff
@@ -0,0 +1,251 @@
+import numpy as np
+import pytest
+from astropy.io import fits
+from astropy.table import Table
+from stdatamodels.jwst.datamodels import ImageModel
+
+from jwst.stpipe import query_step_status
+from jwst.assign_wcs import AssignWcsStep
+from jwst.extract_2d.extract_2d_step import Extract2dStep
+from jwst.master_background import MasterBackgroundMosStep
+from jwst.master_background import nirspec_utils
+
+
+# WCS keywords, borrowed from NIRCam grism tests
+WCS_KEYS = {'wcsaxes': 2, 'ra_ref': 53.1490299775, 'dec_ref': -27.8168745624,
+            'v2_ref': 86.103458, 'v3_ref': -493.227512, 'roll_ref': 45.04234459270135,
+            'v3i_yang': 0.0, 'vparity': -1,
+            'crpix1': 1024.5, 'crpix2': 1024.5,
+            'crval1': 53.1490299775, 'crval2': -27.8168745624,
+            'cdelt1': 1.81661111111111e-05, 'cdelt2': 1.8303611111111e-05,
+            'ctype1': 'RA---TAN', 'ctype2': 'DEC--TAN',
+            'pc1_1': -0.707688557183348, 'pc1_2': 0.7065245261360363,
+            'pc2_1': 0.7065245261360363, 'pc2_2': 1.75306861111111e-05,
+            'cunit1': 'deg', 'cunit2': 'deg'}
+
+
+def create_nirspec_hdul(detector='NRS1', grating='G395M', filter_name='F290LP',
+                        exptype='NRS_MSASPEC', subarray='FULL', slit=None, nint=1,
+                        wcskeys=None):
```

Comment:
Are these functions copied from somewhere?  Can they maybe be imported instead?

Reply:
Yes, they have been, and I've imported create_nirspec_hdul now (and update/add the additional header keywords that are needed).  There is also a bit of overlap for creating the MSA metadata (similar to that in extract_2d.test_nirspec), but this requires a unique set of metadata, so for clarity I've left that here rather than importing and modifying.  Let me know if that sounds reasonable.

Reply:
Yep, looks good.  Thanks!